networks/59141/markets.json

```diff
@@ -0,0 +1,7 @@
+[
+  "spot://ETH/USDC",
+  "spot://WETH/USDC",
+  "spot://WBTC/USDC",
+  "spot://DUCKIES/USDC",
+  "spot://USDC/USDT"
```

Comment:
I think we can add lube as well

Reply:
badbcb5cbf000eadc1f7e12992055a838db1ef84